### PR TITLE
 Add `Rising Global Taiko Showdown 2021` wiki article

### PR DESCRIPTION
**Pull request by fajar13k**
- Wiki article for already ended [Rising Global Taiko Showdown 2021](https://osu.ppy.sh/community/forums/topics/1324349)

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 1]
```diff
@@ -0,0 +1,635 @@
...
+21. If the beatmap ends in a draw, the game will be nullified and the map will be replayed.
+22. In case of a tie in points (example: 4-4 in BO9) the tiebreaker is selected.
+
+### All Stars rules
+
+1. All Stars is a fun event taking place on the same weekend as the Grand Finals! It will showcase the best players of the tournament playing against each other in a special match.
+2. 12 players will participate in this event. They will be decided by a vote.
+3. If there are more than 2 players from a country in the top 12 most voted players, only the 2 best will partake in All Stars.
+4. The top 2 players will be captains of each team. Their teammates will be decided by the ranking of the next 6 players. The topmost voted player will play with players with an odd number and the second most voted player with players with an even number.
+5. The mappool that will be used is the mappool from the Grand Finals stage.
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:590]:
```suggestion
5. The mappool is the same as the one in Grand Finals.
```

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 2]
```diff
@@ -0,0 +1,635 @@
...
+8. Any rule changes or unexpected occurrences will be announced in the Discord server.
+
+### Restrictions and registrations
+
+1. In order to participate, players need to be in the rank range of #1,000 - #4,500.
+2. Players are not allowed to rank up further than #1,000 during the tournament. Passing this limit will result in disqualification.
+3. Players must join the Discord server of the tournament to actually be counted as a player.
+4. Players sign up individually. When their registration is accepted, they will get a player role with their respective country.
+5. osu! staff can refuse players from participating in the tournament if their skill is deemed much higher than the targeted rank range.
+6. In this iteration, 2 teams for a country may be allowed under certain conditions:
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:495]:
```suggestion
6. 2 teams for a country may be allowed under certain conditions:
```

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 3]
```diff
@@ -0,0 +1,635 @@
...
+| **Japan A** ![][flag_JP] | **7** | 4 | ![][flag_AR] Argentina | [#1](https://osu.ppy.sh/community/matches/87180351) |
+
+### Quarterfinals
+
+Saturday, 3 July 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| Argentina ![][flag_AR] | 2 | **6** | ![][flag_CN] **China A** | [#1](https://osu.ppy.sh/community/matches/86713465) |
+| Russian Federation/Ukraine ![][flag_RU] ![][flag_UA] | 4 | **6** | ![][flag_MY] Malaysia A | [#1](https://osu.ppy.sh/community/matches/86713468) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:374]:
```suggestion
| Russian Federation/Ukraine ![][flag_RU] ![][flag_UA] | 4 | **6** | ![][flag_MY] **Malaysia A** | [#1](https://osu.ppy.sh/community/matches/86713468) |
```

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 4]
```diff
@@ -0,0 +1,635 @@
...
+| Group | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| A | New Zealand ![][flag_NZ] | 4 | **5** | ![][flag_MX] **Mexico A** | [#1](https://osu.ppy.sh/community/matches/85849217) |
+| A | Mexico A ![][flag_MX] | 0 | **5** | ![][flag_US] **United States B** | [#1](https://osu.ppy.sh/community/matches/85852346) |
+| B | Singapore B ![][flag_SG] | 0 | **5** | ![][flag_US] **United States A** | [#1](https://osu.ppy.sh/community/matches/85856521) |
+| C | Colombia ![][flag_CO] | 1 | **5** | ![][flag_HK] **Hong Kong** | [#1](https://osu.ppy.sh/community/matches/85856496) |
+| H | Austria ![][flag_AT] | 4 | **5** | ![][flag_CN] **China A** | [#1](https://osu.ppy.sh/community/matches/85866295) |
+| F | China B ![][flag_CN] | 0 | **5** | ![][flag_KR] **South Korea** | [#1](https://osu.ppy.sh/community/matches/85868162) |
+| H | Taiwan ![][flag_TW] | 4 | **5** | ![][flag_AT] **Austria** | [#1](https://osu.ppy.sh/community/matches/85869842) |
+| E | Thailand ![][flag_TH] | 3 | **5** | ![][flag_AU] ![][flag_GU] Australia/Guam | [#1](https://osu.ppy.sh/community/matches/85869850) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:424]:
```suggestion
| E | Thailand ![][flag_TH] | 3 | **5** | ![][flag_AU] ![][flag_GU] **Australia/Guam** | [#1](https://osu.ppy.sh/community/matches/85869850) |
```

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 5]
```diff
@@ -0,0 +1,635 @@
...
+
+### Round of 16
+
+Saturday, 26 June 2021:
+
+| Team 1 |  |  | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **China A** ![][flag_CN] | **6** | 3 | ![][flag_US] United States B | [#1](https://osu.ppy.sh/community/matches/86282475) |
+| **Philippines** ![][flag_PH] | **6** | 1 | ![][flag_SG] Singapore A | [#1](https://osu.ppy.sh/community/matches/86294269) |
+| **Russian Federation/Ukraine** ![][flag_RU] ![][flag_UA] | **6** | 4 | ![][flag_AU] ![][flag_GU] | [#1](https://osu.ppy.sh/community/matches/86297694) |
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:395]: ?

```suggestion
| **Russian Federation/Ukraine** ![][flag_RU] ![][flag_UA] | **6** | 4 | ![][flag_AU] ![][flag_GU] **Australia/Guam** | [#1](https://osu.ppy.sh/community/matches/86297694) |
```

Comment (fajar13k) [reply wiki/Tournaments/GTS/RGTS_2021/en.md:395]: Wdym? Red Team (Left) won the game

Comment (fajar13k) [reply wiki/Tournaments/GTS/RGTS_2021/en.md:395]: Oh, empty text, of course

Comment (Walavouchey) [reply wiki/Tournaments/GTS/RGTS_2021/en.md]: oh right, red team won

**Review by Walavouchey — DISMISSED** [review]

File: wiki/Tournaments/GTS/RGTS_2021/en.md [hunk 6]
```diff
@@ -0,0 +1,635 @@
...
+  - RGTS2021
+  - RGTS
+  - GTS
+---
+
+# Rising Global Taiko Showdown 2021
+
+![RGTS 2021 logo](img/logo.jpg)
+
+The **Rising Global Taiko Showdown 2021** (***RGTS 2021***) was a worldwide double-elimination 2v2 osu!taiko tournament hosted by ![][flag_FR] [Briesmas](https://osu.ppy.sh/users/2865172) ![][flag_FR] [Kasumi-sama](https://osu.ppy.sh/users/6177263), and ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958). Only players ranked between #1,000 and #4,500 could participate. It was the first instalment of the Rising Global Taiko Showdown and part of the Global Taiko Showdown series.
```
Comment (Walavouchey) [inline wiki/Tournaments/GTS/RGTS_2021/en.md:12]:
```suggestion
The **Rising Global Taiko Showdown 2021** (***RGTS 2021***) was a worldwide double-elimination 2v2 osu!taiko tournament hosted by ![][flag_FR] [Briesmas](https://osu.ppy.sh/users/2865172), ![][flag_FR] [Kasumi-sama](https://osu.ppy.sh/users/6177263), and ![][flag_DE] [Sayira](https://osu.ppy.sh/users/7253958). Only players ranked between #1,000 and #4,500 could participate. It was the first instalment of the Rising Global Taiko Showdown and part of the Global Taiko Showdown series.
```

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**